logipack-hub/hub-api/Makefile

```
@@ -0,0 +1,7 @@
+PHONY: build
```

checkmake (0.2.2)

[warning] Target "PHONY" should be declared PHONY.

(phonydeclared)